store/src/sql/entities/event_header.rs

```diff
@@ -0,0 +1,59 @@
+use cid::Cid;
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq, sqlx::Type)]
+pub enum EventType {
+    Init,
+    Data,
+    Time,
+}
+
+#[derive(Debug, Clone)]
+pub struct EventHeaderRow {
+    pub cid: Vec<u8>,
+    pub stream_cid: Vec<u8>,
+    pub event_type: EventType,
+    pub prev: Option<Vec<u8>>,
+}
+
+impl EventHeaderRow {
+    pub fn insert() -> &'static str {
+        "INSERT INTO ceramic_one_event_header (cid, stream_cid, event_type, prev) VALUES ($1, $2, $3, $4)"
+    }
+}
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum EventHeader {
```

Comment:
I don't love using the word "header" here when there is a literal "header" field in events that is different than what this type represents.  Maybe we could use the word "metadata" for this concept instead?  I don't think the protocol/rust-ceramic has a concept of "metadata" yet.  js-ceramic/the aggregated MIDs have metadata, but that's at a higher level, so I think it's okay giving the word a different meaning at this layer for the core protocol.

Reply:
I'm on board with that. I originally named the table `event_metadata` and think I want to rename it back as event_header is unclear.